core/src/com/mygdx/game/managers/BotsManager.kt

```diff
@@ -152,4 +160,12 @@ class BotsManager {
             font.draw(batch, text, miniatureRadius + 10f, baseY.toFloat())
         }
     }
+
+    fun somebodyWon(): Boolean {
+        bots.forEach { bot ->
```

Comment:
tu sie prosi o exists (chyba że takiej funkcji nie ma w kotlinie to wtedy nie xD może nazywać się też `any()`)
return bots.exists { bot -> bot.score >= WIN_SCORE)